linux/meta/heka.yml

```diff
@@ -54,9 +54,178 @@ metric_collector:
         window: 120
         periods: 0
         function: avg
+    linux_system_swap_usage_critical:
+      description: 'There is no more swap free space'
+      severity: critical
+      rules:
+      - metric: swap_free
+        relational_operator: '=='
+        threshold: 0
+        window: 60
+        periods: 0
+        function: max
+    linux_system_swap_activity_warning:
+      description: 'The swap activity is high'
+      severity: warning
+      rules:
+      - metric: swap_io_in
+        relational_operator: '>='
+        threshold: 1048576 # 1 Mb/s
+        window: 120
+        periods: 0
+        function: avg
+      - metric: swap_io_out
+        relational_operator: '>='
+        threshold: 1048576 # 1 Mb/s
+        window: 120
+        periods: 0
+        function: avg
+    linux_system_swap_usage_warning:
+      description: 'The swap free space is low'
+      severity: warning
+      rules:
+      - metric: swap_percent_used
+        relational_operator: '>='
+        threshold: 0.8
+        window: 60
+        periods: 0
+        function: avg
+    linux_system_root_fs_warning:
+      description: "The root filesystem's free space is low"
+      severity: warning
+      rules:
+      - metric: fs_space_percent_free
+        field:
+          fs: '/'
+        relational_operator: '<'
+        threshold: 10
+        window: 60
+        periods: 0
+        function: min
+    linux_system_root_fs_critical:
+      description: "The root filesystem's free space is too low"
+      severity: critical
+      rules:
+      - metric: fs_space_percent_free
+        field:
+          fs: '/'
+        relational_operator: '<'
+        threshold: 5
+        window: 60
+        periods: 0
+        function: min
+    linux_system_other_fs_warning:
+      description: "The filesystem's free space is low"
+      severity: warning
+      no_data_policy: okay
+      rules:
+      - metric: fs_space_percent_free
+        field:
+          fs: '!= /var/lib/nova && != /var/log && != /var/lib/mysql && != / && !~ ceph%-%d+$'
```

Comment:
AFAIK these FS don't exist and can be removed
* /var/lib/nova 
*  /var/log 
* /var/lib/mysql 
* ceph%-%d+$'

Also, I'm wondering if the other_fs alarms must be configured into linux support metadata. This is too specific to the node role and will be overridden in user metadata anyway if we want relevant alarms.